spec/unit/webrtc/callFeed.spec.ts

```diff
@@ -85,5 +85,125 @@ describe("CallFeed", () => {
                 expect(feed.isVideoMuted()).toBeTruthy();
             });
         });
+
+        describe("voice activity detection", () => {
+            it("If voice activity is disabled we should not mute/unmute with it", () => {
```

Comment:
```suggestion
            it("should not mute/unmute if voice activity is disabled", () => {
```

Reply:
The words should make a sentence i.e.: _should not mute/unmute if voice activity is disabled_